seqr/views/apis/users_api.py

```diff
@@ -36,6 +36,21 @@ def get_all_collaborators(request):
     return create_json_response(collaborators)
 
 
+@login_required(login_url=API_LOGIN_REQUIRED_URL)
+@csrf_exempt
+def get_all_analysts(request):
+    analysts = {}
+    for project in get_projects_user_can_view(request.user):
+        analysts.update(_get_project_collaborators(project, include_permissions=False))
+
+    for staff in User.objects.filter(is_staff=True):
+        staff_json = _get_json_for_user(staff)
+        staff_json = {staff_json['username']: staff_json}
+        analysts.update(staff_json)
```

Comment:
So while this works fine, its rare that you want to create an object with one key-value pair, and then call update on another object. Usually, what you would do is directly assign the key-value pair to the final object, or you would create a single object with all the new data and call update once. So for example, you could 
```
for staff in User.objects.filter(is_staff=True):
        staff_json = _get_json_for_user(staff)
        analysts[staff_json['username']] = staff_json
```
or you could do
```
staff_analysts = {staff.username: _get_json_for_user(staff) for staff in User.objects.filter(is_staff=True)}
analysts.update(staff_analysts)
```